.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -51,6 +51,7 @@ This is a list of questions that need to be answered by the contributor in order
  - [ ] High-Level Presentation prepared for Technical Oversight Committee _(~15 mins)_
  - [ ] Are meetings currently held for the project? _(yes / no + details)_
  - [ ] Are meeting minutes, agenda and attendance tracked? _(yes / no + details)_
+ - [ ] Is it a data model? Interested in [FINOS Legend Studio Sandbox](https://www.finos.org/legend)?
```

Comment:
Replaces line 225. This will allow us to know earlier in the process if onboarding onto the FINOS Legend Studio instance is required.